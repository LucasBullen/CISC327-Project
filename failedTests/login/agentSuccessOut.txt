1,3c1,3
< Select mode: atm or agent.
< Logged into agent mode.
< Logged out.
---
> Select mode: atm or agent.
> Logged into agent mode.
> Logged out.
