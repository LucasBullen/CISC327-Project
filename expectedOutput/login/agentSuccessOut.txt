Select mode: atm or agent.
Logged into agent mode.
Logged out.
